users/ignoring.rst

```diff
@@ -140,3 +142,47 @@ all files and directories called "foo", ending in a "2" or starting with
   ``some/directory/`` matches the content of the directory, but not the
   directory itself. If you want the pattern to match the directory and its
   content, make sure it does not have a ``/`` at the end of the pattern.
+
+Ignoring files across all devices
+---------------------------------
+
+Although ``.stignore`` file is not synced between the devices, as mentioned
+previously, it's possible to take an advantage of ``#include`` pattern
+to create a single file that will hold globally ignored patterns.
+
+Assuming the content of the file is::
```

Comment:
Here it's not yet clear what "the file" is. I'd already introduce it here with a name. As for the name I like `.stignore-shared` better than anything with `global`, as it isn't global to all folders, but only shared for a single folder between all devices.

Reply:
Reworded and changed filename.